cli/cyclus.cc

```diff
@@ -240,14 +240,14 @@ int ParseCliArgs(ArgInfo* ai, int argc, char* argv[]) {
       ("restart", po::value<std::string>(),
        "restart from the specified simulation snapshot [db-file]:[sim-id]:[timestep]")
       ("schema",
-       "dump the cyclus master schema including all installed module schemas")
+       "dump the cyclus main schema including all installed module schemas")
```

Comment:
I'm guessing you just did an auto find and replace for changing the default branch name from master to main. Did we also want to update the vocabulary for the schema?

Reply:
I was slightly selective... I thought this was an OK change because it's not a reserved word but a general adjective, but I didn't change the method names in the code because I didn't want to risk breaking something

Reply:
Do we need an adjective for this? I get that there is the core Cyclus schema, but in that sentence we say that it includes the module schemas. So I'm not sure if an adjective is needed here. Nothing that would prevent merging this PR, just an interesting note.

Reply:
I think it makes sense to distinguish the 'main' schema from the 'module' schemas